### ISSUE TITLE
"job.getStatus() === JobStatus.Running" generates Typescript compiler error "TS1362: 'JobStatus' cannot be used as a value because it was exported using 'export type'."

### ISSUE THREAD

**Issue by pvince** [kibertoad/toad-scheduler#77]
Sample code:
```
import { JobStatus, SimpleIntervalJob, Task, ToadScheduler } from 'toad-scheduler';

const scheduler = new ToadScheduler();

const task = new Task('Sample task', () => {
 console.log('I Ran!'); 
});

const job = new SimpleIntervalJob({ seconds: 5 }, task);

scheduler.addSimpleIntervalJob(job);

if (job.getStatus() === JobStatus.RUNNING) {
  console.log('Job is running!');
}
```

This line: `if (job.getStatus() === JobStatus.RUNNING) {` generates the error `TS1362: 'JobStatus' cannot be used as a value because it was exported using 'export type'.`

I think this is caused by the `export type` on this line: https://github.com/kibertoad/toad-scheduler/blob/09a754b5cb5c1b2800a30810d1a9cca30f31022f/index.ts#L5

For now I have worked around this issue by directly referencing JobStatus `import { JobStatus } from 'toad-scheduler/dist/lib/common/Job';`

**Comment by codeboyzhou** [comment]
> Thank you for the report, I'll look into this
@kibertoad Hi, I have a try to fix this issue and committed a new PR #92, it might work, could you please check it when you are free? appreciate :)

**Comment by kibertoad** [comment]
Fixed in 1.6.1
